docs/testingplatform/capabilities.md

```diff
@@ -104,3 +104,15 @@ In conclusion, let's summarize the primary aspects of the capability system:
 * All capabilities should inherit from `ICapability` or an interface that inherits from it, and are exposed through a collection with the `ICapabilities` interface.
 * It aids in the evolution of features without causing breaking changes. If a certain capability is not supported, appropriate action can be taken.
 * The responsibility of designing, shipping, and documenting the usage of a capability lies with the *capability owner*. The testing platform can also *own* a capability in the same way as any other extension.
+
+## Test framework capabilities
```

Comment:
I would have this in a custom file as children of `[Capabilities](capabilities.md)`